Add January updates (#98)
poliastro/poliastro.github.io: content/articles/2022-02-january-updates.md

> Title: December updates

Reviewer: ```suggestion
Title: January updates
```
:sweat_smile:

> Date: 2021-02-01 16:00 CET

Reviewer: ```suggestion
Date: 2022-02-01 16:00 CET
```

Tags: NumFOCUS,Updates
slug: january-updates
lang: en
Author: Juan Luis Cano Rodríguez

> This January we have spent a lot of time discussing Sebastian proposal

Reviewer: ```suggestion
This January, we spent a lot of time discussing Sebastian's proposal
```

to [add a new `OrbitArray` object](https://github.com/poliastro/poliastro/pull/1445)
in line with our Small Development Grant.

> We are now all in the same page and the implementation is progressing nicely!

Reviewer: ```suggestion
We are now all on the same page and the implementation is progressing nicely!
```
